dbuf-core/Cargo.toml

```diff
@@ -4,3 +4,4 @@ edition = "2021"
 version.workspace = true
 
 [dependencies]
+pretty = "0.12.3"
```

Comment:
Btw maybe we might want to move all declarations of dependency versions we use to the workspace `Cargo.toml`? And in crate-level configuration only list dependencies this crate needs and delegate selection of the version to the workspace configuration 
@inttik, what do you think?